lib/pact_broker/client/webhooks/create.rb

```diff
@@ -83,6 +87,10 @@ def webhook_for_consumer_and_provider_url
           "#{pact_broker_base_url.chomp("/")}/webhooks/provider/{provider}/consumer/{consumer}"
         end
 
+        def webhook_with_uuid_for_consumer_and_provider_url
+          "#{pact_broker_base_url.chomp("/")}/webhooks/{uuid}/provider/{provider}/consumer/{consumer}"
```

Comment:
Ideally, we'd want to look up the HAL relation from the index, rather than hardcoding the URL, like the post does. The index would have the following in its response:

{
  "_links": {
     "pb:webhook": {
        "href": "http://broker/webhooks/{uuid}",
        "templated": true
      }
  }
}

and the code would do:

```
index_link.get!._link("pb:webhook").expand(uuid: uuid).put(request_body_with_optional_consumer_and_provider)
```

Reply:
I've created a new helper method `mock_pact_broker_index_with_uuid` which mocks this response - hopefully it it is correct! 🤞